build.js

```diff
@@ -728,57 +661,108 @@ function copyResources() {
         'sourceMappingURL=core.js.map'
     );
 
-    fs.writeFile(corejsPath, result, 'utf8', function(err) {
+    fs.writeFileSync(corejsPath, result, 'utf8', (err) => {
```

Comment:
Putting this after the `await fs.readFile` would be good. Or, if this will all go away by not using core-js, a TODO would be fine.